q2_phylogenomics/tests/test_assemble.py

```diff
@@ -125,6 +131,122 @@ def test_remove_duplicates(self):
             self.assertNotIn('NB501727:157:HFWHJBGXF:3:23610:2922:9731',
                              obs_mapped_ids)
 
+    def test_make_pileups(self):
+        obs_art, = self.plugin.methods['make_pileups'](
+            self.sorted_alignment_maps, self.indexed_genome
+        )
+        obs = obs_art.view(PileUpFilesDirFmt)
+        obs_pileups = obs.pileups.iter_views(PileUpTSVFormat)
+        for _, obs_pileup in obs_pileups:
+            obs_df = pd.read_csv(str(obs_pileup), header=None, sep='\t', )
+            # expected values are derived from running samtools
+            # directly on these input files
+            self.assertEqual(obs_df.shape, (345, 6))
+            self.assertEqual(set(obs_df.iloc[:, 0]), set(['NC_045512.2']))
+
+    def test_consensus_sequence_min_depth_1(self):
+        # this min depth value gets actual sequence with the test data
+        obs_table_art, obs_seq_art = self.plugin.methods['consensus_sequence'](
+            self.pileups, min_depth=1,
+        )
+        obs_table = obs_table_art.view(pd.DataFrame)
+
+        # table tests
+        # two different genomes across four samples
```

Comment:
maybe the test data I built is no good... these were all pulled from a _single genome_ but I just sliced the genome, did not create any overlap (I assumed these would be aligned to the genome with bowtie2 so lack of overlap would be okay and cover the case where we have discontiguous fragments/contigs that fail to align)... why do you see two different genomes? did you make new test data?

Reply:
yep, this was one of the modifications i made. they just differ by one base for the sake of testing.